wiki/middleware.py

```diff
@@ -0,0 +1,27 @@
+import threading
+
+_thread_locals = threading.local()
+
+
+def get_current_request():
+    """Return the request associated with the current thread."""
+    return getattr(_thread_locals, 'request', None)
```

Comment:
@symbolist I do not think we should add an external dependency in `django-wiki`, doing so will also break `testproject` inside this repo.